src/content/troubleshooting/_index.en.md

```diff
@@ -217,11 +217,72 @@ Status:
+trying to access. EndpointSlice resources are created in the same namespace as the source Service. The format of a EndpointSlice
+resource's name is as follows:
+
+`<service-name>--<cluster-id>`
+
+Run `kubectl get endpointslices --all-namespaces |grep <your-service-name>` on the Broker cluster to check if a resource was created for
+your service. If not, then check the Lighthouse Agent logs on the cluster where service was created and look for any error or warning
+messages indicating a failure to create the ServiceImport resource for your service. The most common error is `Forbidden` if the RBAC wasn't
+configured correctly. Depending on the deployment method used, 'subctl' or 'helm', it should've been done for you. Create an
+[issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
+
+If the EndpointSlice resource was created correctly on the Broker cluster, the next step is to check if it exists on the cluster where
+you're trying to access the service. Follow the same steps as earlier to get the list of the EndpointSlice resources and check if the
+EndpointSlice for your service exists. If not, check the logs of the Lighthouse Agent on the cluster where you are trying to access the
+service. As described earlier, it will most commonly be an issue with RBAC otherwise create an
+[issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
+
+If the EndpointSlice resource was created properly on the cluster, run
+`kubectl -n <your-service-namespace> describe endpointslice <your-endpointslice-name>`
+and check if it has the correct endpoint addresses:
+
+```text
+Name:         nginx-ss-cluster2
+Namespace:    default
+Labels:       endpointslice.kubernetes.io/managed-by=lighthouse-agent.submariner.io
+              lighthouse.submariner.io/sourceCluster=cluster2
+              lighthouse.submariner.io/sourceName=nginx-ss
+              lighthouse.submariner.io/sourceNamespace=default
+              multicluster.kubernetes.io/service-name=nginx-ss-default-cluster2
+Annotations:  <none>
+AddressType:  IPv4
+Ports:
+  Name  Port  Protocol
+  ----  ----  --------
+  web   80    TCP
+Endpoints:
+  - Addresses:  10.242.0.5  -----> Pod IP
+    Conditions:
+      Ready:    true
+    Hostname:   web-0   -----> pod hostname
+    Topology:   kubernetes.io/hostname=cluster2-worker2
+  - Addresses:  10.242.224.4
+    Conditions:
+      Ready:   true
+    Hostname:  web-1
+    Topology:  kubernetes.io/hostname=cluster2-worker
+Events:        <none>
+```
+
+If the `Addresses` are correct but still not being returned from DNS queries, try querying IPs in specific cluster using
+by prefixing your service query with `<cluster-id>.` If that returns the IPs correctly, then check the connectivity to the cluster
+using [`subctl show endpoint`](../deployment/subctl/_index.en.md#show-endpoints). The Lighthouse CoreDNS Server only returns IPs
```

Comment:
Don't we want to show the Gateways?

Reply:
`show endpoints` is the one that actually shows connection status